core/src/main/resources/lib/layout/search-bar.jelly

```diff
@@ -38,24 +38,26 @@ THE SOFTWARE.
     @since 2.369
   </st:documentation>
 
-  <div class="jenkins-search ${attrs.clazz}">
-    <div class="jenkins-search__icon">
-      <l:icon src="symbol-search" />
-    </div>
-    <input value="${attrs.value}"
-           id="${attrs.id}"
-           class="jenkins-search__input"
-           placeholder="${attrs.placeholder ?: '%Search'}"
-           type="search"
-           autofocus="${attrs.autofocus == 'true' ? 'true' : null}"
-           autocomplete="off"
-           autocorrect="off"
-           autocapitalize="off"
-           spellcheck="false" />
-    <j:if test="${attrs.hasKeyboardShortcut != 'false'}">
-      <div class="jenkins-search__shortcut" tooltip="${%Press / on your keyboard to focus}">
-        <l:icon src="symbol-search-shortcut" />
+  <div class="jenkins-search-container">
```

Comment:
Why is this a `jenkins-` and not an `app`? Do we expect plugins to copy & paste that? Is this stable API? I assume not, given we're changing a previously `jenkins-` prefixed element structure to support an unrelated UI element.

Reply:
I would expect users to use the search-bar component rather than cobbling it together themselves. It is a fair question though, in this case though any existing implementations with the class ".jenkins-search" would still work as it's just the container that's been added so that the search results are positioned correctly.